File: Exercise/Easy/12. Calculate Perimeter of Rectangle/Code/Calculate Perimeter of Rectangle.cpp
#include <iostream>

using namespace std;

void calcPer(int length, int width) {
    int result = 2 * (length + width);
    cout << result << endl;
}

int main() {
    int length, width;
    cin >> length >> width;

    calcPer(length, width);

    return 0;
}

File: Exercise/Easy/12. Calculate Perimeter of Rectangle/Code/Calculate Perimeter of Rectangle.java
import java.util.Scanner;

public class CalcPer {
    public static void calcPer(int length, int width) {
        int result = 2 * (length + width);
        System.out.println(result);
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int length = scanner.nextInt();
        int width = scanner.nextInt();
        scanner.close();

        calcPer(length, width);
    }
}


